aptos-move/block-executor/src/view.rs

```diff
@@ -1588,21 +1588,6 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>> TModuleView for LatestView
             state_key,
         );
 
-        // Enforce feature gating V2 loader implementation: TModuleView is no longer used in
-        // V2 interfaces because we implement storage traits directly. Use a debug assert to
-        // panic in tests, adn invariant violation for non-debug builds.
-        if self.runtime_environment.vm_config().use_loader_v2 {
```

Comment:
Can we also get rid of the moduleview trait altogether? (or is it in a separate PR?)

Reply:
You see, it is used in prop tests haha

Reply:
So technically we can... there is one place where we call `get_module_bytes` based on the mock key, I haven't figured out an easy way to kill Module vie without killing / changing tests so kept it for now ...

Reply:
We don't even use the MVModuleMap anymore, right? So we can actually remove those tests (ignored already, right), which should simplify the proptests a lot and then we could add new tests together with V2.
I can go ahead and send a PR removing all that once your changes are landed.